phpcs.xml.dist

```diff
@@ -27,6 +27,12 @@
 		<exclude-pattern>tests/</exclude-pattern>
 	</rule>
 
+	<rule ref="PHPCompatibility.FunctionUse.NewFunctionParameters.ucwords_delimitersFound">
```

Comment:
This error is

> The function ucwords() does not have a parameter "delimiters" in PHP version 5.4.31 or earlier

However, WordPress requires `5.6.20+` and WooCommerce lists `7.2+` so I think this is something we can ignore.

Reply:
Woo core still supports 5.6 too fwi. 7.2 is recommended version. You should be able to change the testVersion in the main phpcs file to reflect this. It's 5.4 currently.

Reply:
🤦‍♂. Yeah updating the version is much better. Thanks!